velox/dwio/common/Options.h

```diff
@@ -185,6 +188,20 @@ class RowReaderOptions {
         : std::numeric_limits<uint64_t>::max();
   }
 
+  /**
+   * Set the output type of row reader.
+   */
+  void setOutputType(const RowTypePtr& outputType) {
+    outputType_ = outputType;
+  }
+
+  /**
+   * Get the output type of row reader.
+   */
+  const RowTypePtr& getOutputType() const {
```

Comment:
Requested type is available as `getSelector()->getSchemaWithId()->type`.  We may want to convert it to a type directly in the future, but for now let's not keep 2 copies of the same thing.